src/plugins/intel_cpu/src/emitters/jit_eltwise_emitters.cpp

```diff
@@ -258,6 +259,119 @@ size_t jit_divide_emitter::aux_vecs_count() const {
+    } else {
+        assert(!"unsupported isa");
+    }
+}
+
+template <mkldnn::impl::cpu::x64::cpu_isa_t isa>
+void jit_ceiling_emitter::emit_isa(const std::vector<size_t> &in_vec_idxs, const std::vector<size_t> &out_vec_idxs) const {
+    using Vmm = typename conditional3<isa == cpu::x64::sse41, Xmm, isa == cpu::x64::avx2, Ymm, Zmm>::type;
+    Vmm vmm_src = Vmm(in_vec_idxs[0]);
+    Vmm vmm_dst = Vmm(out_vec_idxs[0]);
+    h->uni_vroundps(vmm_dst, vmm_src, 2);
+}
+
+/// ROUND ///
+jit_round_emitter::jit_round_emitter(jit_generator *host, cpu_isa_t host_isa, const std::shared_ptr<ngraph::Node>& node, Precision exec_prc)
+    : jit_emitter(host, host_isa, node, exec_prc) {
+    const auto round = ngraph::as_type_ptr<ngraph::opset5::Round>(node);
+    assert(round != nullptr);
+
+    const auto node_mode = round->get_mode();
+    assert(node_mode == ngraph::opset5::Round::RoundMode::HALF_AWAY_FROM_ZERO ||
+           node_mode == ngraph::opset5::Round::RoundMode::HALF_TO_EVEN);
+
+    switch (node_mode) {
+        case ngraph::opset5::Round::RoundMode::HALF_AWAY_FROM_ZERO: {
+            mode = RoundMode::HALF_AWAY_FROM_ZERO;
+            break;
+        }
+        case ngraph::opset5::Round::RoundMode::HALF_TO_EVEN: {
+            mode = RoundMode::HALF_TO_EVEN;
+            break;
+        }
+    }
+}
+
+jit_round_emitter::jit_round_emitter(jit_generator* host, cpu_isa_t host_isa, Precision exec_prc)
+    : jit_emitter(host, host_isa, exec_prc) {}
+
+size_t jit_round_emitter::get_inputs_num() const { return 1; }
+
+void jit_round_emitter::emit_impl(const std::vector<size_t>& in_vec_idxs,
+                                    const std::vector<size_t>& out_vec_idxs,
+                                const std::vector<size_t> &pool_vec_idxs, const std::vector<size_t> &pool_gpr_idxs,
+                                const emitter_context *emit_context) const {
+    if (host_isa_ == cpu::x64::sse41) {
+        emit_isa<cpu::x64::sse41>(in_vec_idxs, out_vec_idxs);
+    } else if (host_isa_ == cpu::x64::avx2) {
+        emit_isa<cpu::x64::avx2>(in_vec_idxs, out_vec_idxs);
+    } else if (host_isa_ == cpu::x64::avx512_common) {
+        emit_isa<cpu::x64::avx512_common>(in_vec_idxs, out_vec_idxs);
+    } else {
+        assert(!"unsupported isa");
+    }
+}
+
+template <mkldnn::impl::cpu::x64::cpu_isa_t isa>
+void jit_round_emitter::emit_isa(const std::vector<size_t> &in_vec_idxs, const std::vector<size_t> &out_vec_idxs) const {
+    using Vmm = typename conditional3<isa == cpu::x64::sse41, Xmm, isa == cpu::x64::avx2, Ymm, Zmm>::type;
+    Vmm vmm_src = Vmm(in_vec_idxs[0]);
+    Vmm vmm_dst = Vmm(out_vec_idxs[0]);
+    // Round down: 01B - Rounded result is closest to but no greater than the infinitely precise result.
+    // Round up: 10B - Rounded result is closest to but no less than the infinitely precise result.
+    h->uni_vroundps(vmm_dst, vmm_src, mode == RoundMode::HALF_AWAY_FROM_ZERO ? 1 : 2);
```

Comment:
It's incorrect for these modes.  You should reuse oneDNN emitters for these mods:  [HALF_AWAY_FROM_ZERO](https://github.com/openvinotoolkit/oneDNN/blob/82ca2f931c1d588b67d154d873136d4af1ffb3a8/src/cpu/x64/injectors/jit_uni_eltwise_injector.cpp#L1557) and [HALF_TO_EVEN](https://github.com/openvinotoolkit/oneDNN/blob/82ca2f931c1d588b67d154d873136d4af1ffb3a8/src/cpu/x64/injectors/jit_uni_eltwise_injector.cpp#L1551) as [Gelu-7](https://github.com/openvinotoolkit/openvino/blob/4b29eed013e1e86bab079831b5be7f126890daba/src/plugins/intel_cpu/src/emitters/jit_mkldnn_ext_emitters.hpp#L127) for example.

Reply:
done, thanks